fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/socialmedia/NotifyAddChirp.java

```diff
@@ -19,7 +19,7 @@
    * @param channel channel where the post is located
    * @param timestamp UNIX timestamp in UTC
    */
-  public AddChirpBroadcast(String postId, String channel, long timestamp) {
+  public NotifyAddChirp(String postId, String channel, long timestamp) {
```

Comment:
The toString of NotifyAddChirp should be updated too, it's still using "AddChirpBroadcast"